interface/src/app/gene/network/network.js

```diff
@@ -187,20 +205,22 @@ angular.module('adage.gene.network', [
           NodeService.get(
             {'heavy_genes': heavyGenes, 'limit': 0},
             function success(response) {
-              var n = response.objects.length;
+              var i = 0, n = response.objects.length;
+              var uri;
               str += '<br />' + n + (n > 1 ? ' nodes are ' : ' node is ');
               str += 'related to both genes' + (n > 0 ? ':' : '.');
-              for (var i = 0; i < n; ++i) {
-                str += '<br />* ' + response.objects[i].name + '</li>';
+              for (; i < n; ++i) {
+                uri = '<a href="#/node/' + response.objects[i].id + '">';
+                str += '<br />* ' + uri + response.objects[i].name + '</a>';
```

Comment:
One more comment here on the subject of tension between DOM manipulation and Angular and then I'll stop harping on it: if this code were written in a more Angular-compatible way, we could use Angular's excellent HTML template features to format this tip.

I suspect this level of integration of Angular with d3 to be somewhat tricky, so perhaps it's not practical to do it that way for now.